src/components/BattleTV/BattleTV.module.css

```diff
@@ -1,192 +1,189 @@
-  font-weight: 600;
-  color: #ffffff;
+  padding-top: 10px;
+  font-family: 'Red Hat Display', sans-serif;
+  margin: auto;
 }
 
-.battlecardwin {
-  background-color: #00bc6244;
-}
-.battlecardlose {
-  background-color: #bc170044;
+.list {
+  margin: 0 auto;
+  background: transparent;
+  box-shadow: 0 0 4rem 0 rgba(0, 0, 0, 0.1),
+    0 1rem 2rem -1rem rgba(0, 0, 0, 0.3);
+  text-align: center;
+  vertical-align: middle;
+  border: none;
+  border-spacing: 0px 15px;
+  border-collapse: separate;
 }
-.battlecardtie {
-  background-color: #808080;
+
+.item {
+  border-radius: 10px;
+  background: rgba(30, 30, 30, 0.87);
+  max-width: 900px !important;
 }
 
-.battlecard img {
-  width: 80%;
-  /* height: 4rem; */
+.list .item .pic {
+  width: 2rem;
+  height: 2rem;
+  align-items: center;
   border-radius: 50%;
-  background-size: cover;
-  background-position: center;
   box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.2),
     0 1rem 1rem -0.5rem rgba(0, 0, 0, 0.3);
+  border: none;
 }
 
-.pic {
-  width: 5%;
-  text-align: center;
+.list .item .watchButton {
+  transform: scale(0.8);
 }
 
-.username {
-  width: 17%;
-  font-size: 1.3vw;
-  line-height: 24px;
+.list .item .watchButton:hover {
+  transform: scale(1);
 }
 
-.left {
-  text-align: left;
+.battlecardwin {
+  outline: thin solid #00bc62;
+  box-shadow: 0 3px #ffe600;
 }
 
-.right {
-  text-align: right;
+.battlecardlose {
+  outline: thin solid #bc1700;
 }
 
-.coinused {
-  width: 7%;
-  text-align: center;
-  font-size: 1vw;
-  line-height: 22px;
+.battlecardtie {
+  outline: thin solid #ffe600;
+  max-width: 900px !important;
 }
-.destruction {
-  width: 12%;
-  text-align: center;
-  font-size: 2vw;
-  line-height: 25px;
+
+.list .item .name {
+  flex-basis: 6rem;
+  font-size: 1.1rem;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  color: white;
+  margin-right: 5px;
+  border: none;
 }
-.vs {
-  text-align: center;
-  font-size: 1vw;
-  line-height: 22px;
-}
-.watchButton {
-  font-size: 1vw;
-  background-color: #4595cf;
-  padding: 5px;
-  border-radius: 4px;
-  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.2);
-  cursor: pointer;
-  transition: all 0.05s ease-in-out;
-  user-select: none;
-}
-.watchButton:hover {
-  background-color: #529dd3;
-  box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.2),
-    0 1rem 1rem -0.5rem rgba(0, 0, 0, 0.3);
+
+.list .item .score {
+  font-family: monospace;
+  color: white;
+  background-color: transparent;
+  margin-right: 1.5rem;
+  border: none;
 }
-.watchButton:active {
-  background-color: #4595cfaa;
-  box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.2),
-    0 1rem 1rem -0.5rem rgba(0, 0, 0, 0.3);
+
+.list .item .score:after {
+  margin-right: 1rem;
+  opacity: 0.8;
+  background-color: transparent;
+  border: none;
+}
+
+.list .item .score:first-child {
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+}
+
+.list .item .score:last-child {
+  border-bottom-right-radius: 10px;
+  border-top-right-radius: 10px;
 }
```

Comment:
What does these do exactly? .

Reply:
Gets the corners to be rounded for the row